src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
can't we use `LTarget,Operand,RTarget`  which is way more natural to scan 

also, how do you add multiple constraints? using `;` as sep?